modules/lvgl/lvgl.c

```diff
@@ -244,9 +244,12 @@ static int lvgl_init(void)
 	return 0;
 }
 
-BUILD_ASSERT(CONFIG_APPLICATION_INIT_PRIORITY < CONFIG_LV_Z_INPUT_INIT_PRIORITY);
+/* Display device needs to be ready */
+BUILD_ASSERT(CONFIG_KERNEL_INIT_PRIORITY_DEVICE < CONFIG_LV_Z_SYS_INIT_PRIORITY);
```

Comment:
`CONFIG_DISPLAY_INIT_PRIORITY`. Note that (hopefully) device init priorities will disappear soon.